backend/loaders/ppt_loader.py

```diff
@@ -0,0 +1,26 @@
+from pptx import Presentation
+import os
+
+def load_presentation(presentation_file: bytes, filename: str, file_meta: UploadFile) -> str:
+    presentation = Presentation(presentation_file)
+    text_content = ""
+
+    for slide in presentation.slides:
+        slide_text = ""
+        for shape in slide.shapes:
+            if shape.has_text_frame:
+                for paragraph in shape.text_frame.paragraphs:
+                    for run in paragraph.runs:
+                        slide_text += run.text.strip()
+        text_content += slide_text.replace("\n", " ")
+
+
+    # Write files to misc folder
+    misc_dir = os.path.join(os.getcwd(), "misc")
+    filename = filename.split(".")[0]  #to remove the .ppt extension from the filename

```

Comment:
@grumpyp is this necessary? I had noticed this was done in one of the other loaders but don't see it in the main code now. 
Will the function expect filename without the extension?

Reply:
Please build it like the other loaders. We do not cut any fileextension off there.